lib/liquid/tag.rb

```diff
@@ -13,7 +13,15 @@ def parse(tag_name, markup, tokenizer, parse_context)
         tag
       end
 
+      def disable_nested_tags(*tags)
+        @disabled_tags = tags
```

Comment:
Should we do `disabled_tags << tags` so the method can be called multiple times?

Reply:
Makes sense